file_system.py

```diff
@@ -0,0 +1,46 @@
+import csv
+import re
+
+from spending import Spending
+
+
+class FileManipulator:
```

Comment:
its better to pass filename to constructor and use it further to make it universal